notebooks/unit_test_prohibited_apps_spawning_cmdprompt.ipynb

```diff
@@ -0,0 +1,204 @@
+       "      <td>eyJlbnRpdHlUeXBlIjoiREVWSUNFIiwicHJpbWFyeUFydG...</td>\n",
+       "      <td>winword.exe</td>\n",
+       "      <td>[eyJlbnRpdHlUeXBlIjoiREVWSUNFIiwicHJpbWFyeUFyd...</td>\n",
+       "      <td>cmd.exe</td>\n",
+       "      <td>2020-10-06 17:13:29</td>\n",
+       "      <td>eyJlbnRpdHlUeXBlIjoiVVNFUiIsInByaW1hcnlBcnRpZm...</td>\n",
+       "      <td>c:\\program files\\microsoft office\\winword.exe</td>\n",
+       "      <td>TBD</td>\n",
+       "      <td>{\"process_path\":\"c:\\\\windows\\\\system32\",\"proce...</td>\n",
+       "      <td>{'process_path': 'c:\\windows\\system32', 'proce...</td>\n",
+       "      <td>2020-10-06 17:13:29</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "           start_time  ...           timestamp\n",
+       "0 2020-10-06 17:13:29  ... 2020-10-06 17:13:29\n",
+       "\n",
+       "[1 rows x 12 columns]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<spl2_kernel.spl2_runner.SPL2Job at 0x7fd58e124e50>"
+      ]
+     },
+     "execution_count": 94,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "| from read_text(\"s3://smle-experiments/datasets/ssa/detect_prohibited_applications_spawning_cmd_exe.json\")\n",
+    "| eval input_event=from_json_object(value)\n",
+    "\n",
+    "| eval timestamp=parse_long(ucast(map_get(input_event, \"_time\"), \"string\", null))\n",
+    "| eval process_name=ucast(map_get(input_event, \"process_name\"), \"string\", null),\n",
+    "parent_process=lower(ucast(map_get(input_event, \"parent_process_name\"), \"string\", null)),\n",
+    "dest_user_id=ucast(map_get(input_event, \"dest_user_id\"), \"string\", null),\n",
+    "dest_device_id=ucast(map_get(input_event, \"dest_device_id\"), \"string\", null)\n",
+    "| where process_name=\"cmd.exe\"\n",
+    "| rex field=parent_process \"(?<field0>[^\\\\\\\\]+)$\"\n",
+    "| where field0=\"winword.exe\" OR\n",
+    "        field0=\"excel.exe\" OR\n",
+    "        field0=\"outlook.exe\" OR\n",
+    "        field0=\"powerpnt.exe\" OR\n",
+    "        field0=\"visio.exe\" OR\n",
+    "        field0=\"mspub.exe\" OR\n",
+    "        field0=\"acrobat.exe\" OR\n",
+    "        field0=\"acrord32.exe\" OR\n",
+    "        field0=\"chrome.exe\" OR\n",
+    "        field0=\"iexplore.exe\" OR\n",
+    "        field0=\"opera.exe\" OR\n",
+    "        field0=\"firefox.exe\" OR\n",
+    "        field0=\"java.exe\" OR\n",
+    "        field0=\"powershell.exe\"\n",
+    "| eval start_time=timestamp,\n",
+    "end_time=timestamp,\n",
+    "entities=mvappend(dest_device_id, dest_user_id),\n",
+    "body=\"TBD\";"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Notebook runs great there is one empty cell looks minor but couldnt quite make out what the purpose of this block was most likely boiler plate